back/config/locales/en.yml

```diff
@@ -177,6 +177,8 @@ en:
         description:
       other_input_field:
         title: Type your answer
+      ask_follow_up_field:
+        title: Tell us why (optional)
```

Comment:
Do we need '(optional)' - shouldn't the schema add that in automatically?

Reply:
I don't think the schema adds it, but I can add it manually on the FE instead to keep this cleaner :+1: